Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -362,6 +380,26 @@ - (void)primeSyncManagerWithSemaphore:(dispatch_semaphore_t)semaphore {
     }];
 }
 
+- (void)disableNetworking {
+    NSTask *task = [[NSTask alloc] init];
+    task.launchPath = @"/bin/bash";
+    task.currentDirectoryPath = NSProcessInfo.processInfo.environment[@"PWD"];
+    task.arguments = @[@"./scripts/disable_networking.sh"];
+    [task launch];
+    [task waitUntilExit];
+}
+
+- (void)enableNetworkingAfter:(NSTimeInterval)secs {
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(secs * NSEC_PER_SEC)), dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
+        NSTask *task = [[NSTask alloc] init];
+        task.launchPath = @"/bin/bash";
+        task.currentDirectoryPath = NSProcessInfo.processInfo.environment[@"PWD"];
```

Comment:
Should this instead use `-[NSFileManager currentDirectoryPath]` rather than using the value of the environment variable? I don't think there's any guarantee that `PWD` is set to a meaningful value.